test/metabase/upload_test.clj

```diff
@@ -863,13 +864,14 @@
                                           "9000000000,Razor Crest,Din Djarin,Spear"])
                     :auxiliary-sync-steps :synchronous))]
           (testing "Check the data was uploaded into the table correctly"
-            (is (= [@#'upload/auto-pk-column-name "id" "ship" "name" "weapon"]
+            (is (= (header-with-auto-pk ["id" "ship" "name" "weapon"])
                    (column-names-for-table table)))
             (is (=? {:name                       #"(?i)id"
                      :semantic_type              :type/PK
                      :base_type                  :type/BigInteger
                      :database_is_auto_increment false}
-                    (t2/select-one Field :database_position 1 :table_id (:id table))))))))))
+                    (let [pos (if (#'upload/auto-pk-column? driver/*driver* (mt/db)) 1 0)]
```

Comment:
hehe, usually, you're the one to point out reusable piece of code so I wonder why you don't put this into a helper function?

Reply:
Fair point, but CI just finished 😆 

I really should take my own medicine though.